xpk.py

```diff
@@ -1373,12 +1418,15 @@ def enable_kueue_crds(args, system) -> int:
   Returns:
     0 if successful and 1 otherwise.
   """
-  cluster_hardware_name = f'{args.num_slices}x{args.tpu_type}'
+  device_type, acc_type = get_device_and_accelerator_type(args)
```

Comment:
Nit Lets use accelerator_type instead of acc_type.

Reply:
This is also a case where we recheck the device type and accelerator type. We can get this info directly from `system` that is passed into the `enable_kueue_crds`

Reply:
we can't know device_type because we only pass system_characteristics. I will make a change to pass device_type to enable_kueue_crds with args and system